### DISCUSSION TITLE
Assign random reviewer after moving from draft to open

### DISCUSSION THREAD

**Discussion by marplatense** [community/community#113327] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

I have a codeowner team and I assign random reviewers via load balance algorithm,
* If I create a PR in draft mode, system does not assign a reviewer yet.
* While the PR still in draft, I assign a reviewer who is part of the codeowner team.
* I move the PR from draft to open
* The system does assign a new reviewer, since I already have one there. Therefore I was able to play the system and chose the reviewer myself.

Is there a way to change this behavior? (so that a new random reviewer is picked when I open the PR)

**Comment by vineeth-au** [comment]
I guess a [codeowners file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) might help you...

PS: Adding [branch protection rule](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-pull-request-reviews-before-merging) also allows you to require reviews before merging.

**Comment by marplatense** [comment]
Thank you @vineeth-au for your answer. 
I have a codeowners file (otherwise I believe it is impossible to set random reviewers). Regarding the second suggestion, this person (assigned beforehand and not random at all) is a reviewer so I don't see how a branch protection would help me here.

**Comment by marplatense** [comment, marked as answer]
Answer was to disable the " Count existing requests " option.
